### PR TITLE
fix/TR-43-adaptieve-size-for-checkboxes

### PR DESCRIPTION
**Pull request by meefox**
Related to https://oat-sa.atlassian.net/browse/TR-43

- add vertical-align for content when font size is smaller than a row
- add vertical-align for list item when font size is smaller than a row
- fix font-size for controls in headers when custom font size is applied
- fix font-size for delete icon when custom font size is applied
- fix position for delete icon when custom font size is applied

### CODE REVIEW

**Review by jsconan — COMMENTED** [review]
As the SCSS has been change, could you also compile them and commit the result CSS?
```
cd tao/views/build
npx grunt taoqtiitemsass
```

I tested it, but the issue is still there. Did I miss something?

**Review by jsconan — COMMENTED** [review]
It works well for small font size. It does not for larger size.
This fixes the particular use case mentioned in the ticket though.

The target branch is `master`, could you rebase the PR to target `develop` instead?

File: views/scss/qti/_choice.scss [hunk 1]
```diff
@@ -45,38 +52,18 @@
...
+            //both label and checkbox
+            [data-eliminable] {
+                display: block;
+            }
         }
         // eliminator outer layer
         .qti-choice {
             // when checked
             &.eliminated {
+                background: white(.7);
```
Comment (jsconan) [inline views/scss/qti/_choice.scss:65]: **nitpick:** Please use variables for the colors. There is a list available here: https://github.com/oat-sa/tao-core-ui-fe/blob/master/scss/inc/_colors.scss
This is important for theming.

**Review by jsconan — COMMENTED** [review]
It looks ok on Firefox, but not on other browsers like Chrome, Edge or IE: the checkbox is slightly moved down.

Default font size
image

Small font size
image

Huge font size
image

**Review by jsconan — APPROVED** [review]
Well done!

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Comment by btamas** [comment]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Comment by btamas** [comment]
Is this acceptable? If yes, it is approved.
Screen Shot 2020-11-04 at 11 06 35

**Comment by meefox** [comment]
thank you @btamas good catch.
No, it is not, will update it.

**Comment by meefox** [comment]
Screenshot 2020-11-04 at 21 11 57
This is the best that can I achieve with the current layout and playing save

**Comment by btamas** [comment]
> This is the best that can I achieve with the current layout and playing save

It is good for me.
- Please merge develop
- Recompile css if it is necessary
- and I can approve and merge it.

**Review by btamas — APPROVED** [review]

**Pull request merged**